tests/packagedcode/data/nuget/packages.lock.json.expected

```diff
@@ -0,0 +1,3068 @@
+[
+  {
+    "type": "nuget",
+    "namespace": null,
+    "name": null,
+    "version": null,
+    "qualifiers": {},
+    "subpath": null,
+    "primary_language": null,
+    "description": null,
+    "release_date": null,
+    "parties": [],
+    "keywords": [],
+    "homepage_url": null,
+    "download_url": null,
+    "size": null,
+    "sha1": null,
+    "md5": null,
+    "sha256": null,
+    "sha512": null,
+    "bug_tracking_url": null,
+    "code_view_url": null,
+    "vcs_url": null,
+    "copyright": null,
+    "holder": null,
+    "declared_license_expression": null,
+    "declared_license_expression_spdx": null,
+    "license_detections": [],
+    "other_license_expression": null,
+    "other_license_expression_spdx": null,
+    "other_license_detections": [],
+    "extracted_license_statement": null,
+    "notice_text": null,
+    "source_packages": [],
+    "file_references": [],
+    "is_private": false,
+    "is_virtual": false,
+    "extra_data": {
+      "target_framework": "net6.0"
+    },
+    "dependencies": [
+      {
+        "purl": "pkg:nuget/McMaster.Extensions.Hosting.CommandLine@4.0.1",
+        "extracted_requirement": "[4.0.1, )",
+        "scope": "net6.0",
+        "is_runtime": true,
+        "is_optional": false,
+        "is_resolved": true,
+        "is_direct": true,
+        "resolved_package": {
+          "type": "nuget",
+          "namespace": null,
+          "name": "McMaster.Extensions.Hosting.CommandLine",
+          "version": "4.0.1",
+          "qualifiers": {},
+          "subpath": null,
+          "primary_language": null,
+          "description": null,
+          "release_date": null,
+          "parties": [],
+          "keywords": [],
+          "homepage_url": null,
+          "download_url": null,
+          "size": null,
+          "sha1": null,
+          "md5": null,
+          "sha256": null,
+          "sha512": null,
+          "bug_tracking_url": null,
+          "code_view_url": null,
+          "vcs_url": null,
+          "copyright": null,
+          "holder": null,
+          "declared_license_expression": null,
+          "declared_license_expression_spdx": null,
+          "license_detections": [],
+          "other_license_expression": null,
+          "other_license_expression_spdx": null,
+          "other_license_detections": [],
+          "extracted_license_statement": null,
+          "notice_text": null,
+          "source_packages": [],
+          "file_references": [],
+          "is_private": false,
+          "is_virtual": true,
+          "extra_data": {},
+          "dependencies": [
+            {
+              "purl": "pkg:nuget/McMaster.Extensions.CommandLineUtils@4.0.1",
+              "extracted_requirement": "4.0.1",
+              "scope": "net6.0",
+              "is_runtime": true,
+              "is_optional": false,
+              "is_resolved": true,
+              "is_direct": false,
```

Comment:
```suggestion
              "is_direct": true,
```

`is_direct: True` is not only for the direct dependencies of the main root package, but also for all direct dependency relationships of transitive dependencies.